bin/exporter

```diff
@@ -16,10 +16,12 @@ blacklisted_tables = ENV.fetch "BLACKLISTED_TABLES", ""
 whitelisted_tables = ENV.fetch "WHITELISTED_TABLES", ""
 blacklisted_fields = ENV.fetch "BLACKLISTED_FIELDS", ""
 
+db_pass = ENV.fetch( 'DB_PASS', File.open(ENV.fetch( 'DB_CRED_FILE' ), &:readline))
+
 client    = ConnectionPool::Wrapper.new(size: 15, timeout: 3) do
               Mysql2::Client.new( host: ENV.fetch( "DB_HOST" ),
                               username: ENV.fetch( "DB_USER" ),
-                              password: ENV.fetch( "DB_PASS" ),
+                              password: db_pass,
```

Comment:
should the cred file contain both the password _and_ the username?

Reply:
I can add the username as well